packages/docs/docs/concepts/progressive-loading/volumeProgressive.md

```diff
@@ -0,0 +1,134 @@
+```
+
+Any other tools creating multipart/related encapsulated data can be used, as
+can using accept headers or parameters for a standard DICOMweb server.
+
+Note the data path for these is, in general the normal DICOMweb path with
+`/frames/` replaced by some other name.
+
+# Configuration
+
+See the volumeProgressive example for full details.
+
+No special configuration is required for HTJ2K images streaming, but that isn't
+particularly effective. The byte range request is preferably, but requires
+additional server side support for byte ranges. Other types require custom
+configuration.
+
+## HTJ2K Byte Range Configuration
+
+```javascript
+cornerstoneDicomImageLoader.configure({
+  retrieveOptions: {
+    default: {
+      '3.2.840.10008.1.2.4.96': {
+        streamingDecode: true,
+      },
+      default: {},
+    },
+    multipleFinal: {
+      default: {
+        range: 1,
+      },
+    },
+    multipleFast: {
+      default: {
+        // Note it is the decode that is streaming, but a range request to
+        // get the data, not a streaming request.
+        streamingDecode: true,
+        range: 0,
+        initialBytes: 64000,
+        decodeLevel: 0,
+      },
+    },
+  },
+```
+
+# Performance
+
+The performance gains on this vary quite a bit depending on size of data
+and capabilities of the DICOMweb server components. In general, a 350% speed improvement
+to first volume render is seen because of the interleaved images filling
+nearby images. For straight JLS versus HTJ2K, that depends on the compression
+size - JLS compresses a bit better than HTJ2K, resulting in faster downloads.
+However, HTJ2K is faster decoding, resulting in lower times from download
+until image rendering.
+
+For reduced resolution images, both the HTJ2K and JLS versions are slower
+to final image data than the original versions because there are additional
+requests required to fetch all data. HTJ2K decoding still has some decompression
+issues on very small thumbnails resulting in some issues decoding small
+reduced resolution versions. That results in higher time to first volume times.
+
+Note that none of the times include time to load the decoder, which can be
+a second or more, but is only seen on first render. These times are similar for
+both types.
+
+Full size images are 512x512, and JLS reduced ones are 128x128. Note that
+JLS reduced images are JLS lossy compressed in addition to sub resolution.
+
+| Type             | Size  | Network | First Render | Final Render |
+| ---------------- | ----- | ------- | ------------ | ------------ |
+| JLS              | 30 M  | 4g      | 2265 ms      | 8106 ms      |
+| JLS Reduced      | 3.6 M | 4g      | 1690 ms      | 8455 ms      |
+| HTJ2K            | 33 M  | 4g      | 2503 ms      | 8817 ms      |
+| HTJ2K Byte Range | 11.1M | 4g      | 985 ms       | 8786 ms      |
+
```

Comment:
same comment about the necessity of extra column that gives intial render (no progressive) time

Reply:
Those are just the "Final Render" times for the JLS and HTJ2K - within 100 ms anyways.

Reply:
Final render => baseline